test/unit/lib/tasks/reslugging_test.rb

```diff
@@ -2,15 +2,75 @@
 require "rake"
 
 class ResluggingTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
   teardown do
+    task.reenable
```

Comment:
sidenote, I'm not really sure why it needs to be reenabled each time but if it isn't then it doesn't work!